web/src/main/java/it/pagopa/selfcare/onboarding/web/model/mapper/OnboardingInstitutionInfoMapper.java

```diff
@@ -21,8 +21,8 @@
 @Mapper(componentModel = "spring")
 public interface OnboardingInstitutionInfoMapper {
     @Mappings({
-            @Mapping(target = "institution", expression = "java(toDataImpl(model.getInstitution()))"),
-            @Mapping(target = "geographicTaxonomies", expression = "java(mapGeographicTaxonomies(model.getGeographicTaxonomies()))")
+            @Mapping(target = "geographicTaxonomies", expression = "java(mapGeographicTaxonomies(model.getGeographicTaxonomies()))"),
```

Comment:
you could use "uses" attribute for indicate others mappers....look at https://www.baeldung.com/java-mapstruct-nested-mapping

Reply:
Here im using a default method for the mapping, its not referring to another mapper